### DISCUSSION TITLE
Unread email/notification/unread email count added to tray icon

### DISCUSSION THREAD

**Discussion by Wrizz** [tutao/tutanota#2771] [Ideas]
As the title says. It would be cool to have some kind of indicator on the icon in the tray that shows when you have unread email.
I assume due to security reasons it would be better to not make it show the unread email count. 
So just an indicator to show that there is at least one unread email would be great.

You might say that there is a notification when you receive an email. Well, yes, but it goes away after some seconds and that is it. I can't see it when I am away from pc. 

Just a small idea.

**Comment by alstr** [comment]
I have been thinking about the unread emails count and would really find it useful if they synced across platform: i.e., if I read an email on desktop, the unread count (for the app/tray icon) should go down on mobile. Just saves opening the app to clear the count and would help see whether you actually have any new emails. I'm guessing there are technical hurdles here though.
